src/test/java/com/wikia/webdriver/common/contentpatterns/MercurySubpages.java

```diff
@@ -31,16 +31,16 @@ private MercurySubpages() {}
   public static final String CC_MAIN_PAGE = "Mercury_CC_Wikia";
 
   // Categories in mercurycc.wikia.com
-  public static final String CC_CATEGORY_ARTICLES = "wiki/Mercury_CC_Wikia/main/category/Articles";
-  public static final String CC_CATEGORY_BLOGS = "wiki/Mercury_CC_Wikia/main/category/Blog_posts";
-  public static final String CC_CATEGORY_TEMPLATES = "wiki/Mercury_CC_Wikia/main/category/Templates";
-  public static final String CC_CATEGORY_QWERTY = "wiki/Mercury_CC_Wikia/main/category/Qwerty";
-  public static final String CC_CATEGORY_28_ITEMS = "wiki/Mercury_CC_Wikia/main/category/28itemsCategory";
-  public static final String CC_CATEGORY_10_ITEMS = "wiki/Mercury_CC_Wikia/main/category/10itemsCategory";
+  public static final String CC_CATEGORY_ARTICLES = "main/category/Articles";
+  public static final String CC_CATEGORY_BLOGS = "main/category/Blog_posts";
+  public static final String CC_CATEGORY_TEMPLATES = "main/category/Templates";
+  public static final String CC_EMPTY_CATEGORY = "main/category/Qwerty";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)